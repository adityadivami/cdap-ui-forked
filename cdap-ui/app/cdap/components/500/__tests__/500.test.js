import React from 'react';
import { render } from 'react-testing-library';
import Page500 from 'components/500';

describe('Page500', () => {
  it('renders correctly', () => {
    const { container } = render(<Page500 />);
    expect(container.firstChild).toMatchSnapshot();
  });
});

