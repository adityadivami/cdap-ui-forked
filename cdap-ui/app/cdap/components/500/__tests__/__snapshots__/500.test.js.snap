// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page500 renders correctly 1`] = `
<div
  class="page-500"
>
  <h1
    class="error-main-title"
  >
    features.Page500.mainTitle
  </h1>
  <h1>
    <strong>
      features.Page500.secondaryTitle
    </strong>
  </h1>
  <div
    class="message-section"
  >
    <h4>
      <strong>
        features.Page404.subtitleMessage1
      </strong>
    </h4>
    <div
      class="navigation-section"
    >
      <div>
        features.Page500.suggestion1Part1
        <a
          href="http://localhost/"
        >
          features.Page500.suggestion1Part2
        </a>
        features.Page500.suggestion1Part3
      </div>
      <div>
        features.Page500.suggestion2
      </div>
    </div>
  </div>
</div>
`;
